victoria_sensor_msgs/msg/DistanceDisplacement1D.msg

```diff
@@ -0,0 +1,9 @@
+# This message represents the data from a 1D sensor
+# that measures distance it is displaced. It has a
+# min and max value with a current
+# value. All values are meters.
+
+std_msgs/Header header
+float64 current_value
```

Comment:
Can we make this `displacement` or something like that?
That way, when used it will read like `msg.displacement` instead of `msg.current_value`